cuML/CMakeLists.txt

```diff
@@ -120,14 +124,18 @@ add_library(cuml SHARED
             src/dbscan/dbscan.cu
             src/kmeans/kmeans.cu
             src/glm/glm.cu
+	    	src/knn/knn.cu
             )
 
-set(CUML_LINK_LIBRARIES ${CUDA_cublas_LIBRARY} ${CUDA_curand_LIBRARY} ${CUDA_cusolver_LIBRARY} ${CUDA_CUDART_LIBRARY} z)
-if(OPENMP_FOUND)
-    set(CUML_LINK_LIBRARIES ${CUML_LINK_LIBRARIES} OpenMP::OpenMP_CXX pthread)
-endif(OPENMP_FOUND)
-
-target_link_libraries(cuml ${CUML_LINK_LIBRARIES})
+target_link_libraries(cuml
```

Comment:
why remove the check of `OPENMP_FOUND`? I think it needs to be back (though I really like the non single line formatting here)

Reply:
To be quite honest, I'm not sure why this removed. I don't remember removing it but it's possible I had been in the wrong branch when I was messing around with OPENMP last week.